php/Factory.php

```diff
@@ -15,64 +12,23 @@
 class Factory
 {
     /**
-     * @return ContainerBuilder
-     */
-    public function createDIContainer($configFileName, $configDir, $additionalParameters = [])
-    {
-        $container = new ContainerBuilder;
-
-        $loader = new YamlFileLoader($container, new FileLocator($configDir));
-
-        $loader->load($configFileName);
-
-        foreach ($additionalParameters as $name => $value) {
-            $container->setParameter($name, $value);
-        }
-
-        $container->compile();
-
-        return $container;
-    }
-
-    /**
-     * @param string $implementation absolute namespace path to a concrete logger
-     * @param array $parameters parameters to logger's constructor
+     * @param string $path the log file path
      * @return \Psr\Log\LoggerInterface
      */
-    public function createLogger($implementation, $parameters = [])
+    public static function createLogger($path, $is_null = false)
```

Comment:
Static methods are not testable or at least hard to test. Additionally they tempt potential developer to use it from inside application, for convenience ;)